apps/system/js/accessibility.js

```diff
@@ -309,12 +331,19 @@
           return;
       }
 
-      this.speak(aDetails.data, null, {
+      var hintCallback = function ar_hintCallback() {
+        if (options.hints) {
+          this.setHintsTimeout(options.hints);
```

Comment:
So I think we want to combine aDetails.data with options.hints , so instead of 
'Some link' timeout then 'Double tap to activate' we hear:
'Some link' timeout then 'Some link double tap to activate'